specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/examples/AvailableDelegationsResourceGroupGet.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters" : {
+    "api-version": "2018-08-01",
+    "location": "regionName",
+    "subscriptionId" : "subId",
+    "resourceGroupName" : "rg1"
+  },
+  "responses" : {
+    "200" : {
+      "body" : {
+        "value": [
+          {
+            "name": "Microsoft.Provider.resourceType",

```

Comment:
"name", "id", "type" are not properties of "AvailableDelegation"

Reply:
fixed.